README.md

```diff
@@ -187,6 +187,10 @@ Right now the language server supports the following actions:
   "img": "https://example.com/images/golang-intro.png"
   }
   ```
+- `CliConfigSettingsSastEnabled` triggers the api call to check if Snyk Code is enabled
```

Comment:
Proposal: Let's remove the "CLIConfig" Prefix, which is an implementation detail and doesn't add any additional meaning.

Reply:
sure